vj4/handler/base.py

```diff
@@ -502,13 +503,16 @@ def decorate(coro):
 def sanitize(func):
   @functools.wraps(func)
   def wrapped(self, **kwargs):
+    accept_kwargs = inspect.getfullargspec(func).varkw != None
+    new_kwargs = {}
```

Comment:
We should delete in `KeyError` instead of creating a new `dict` because 1) `KeyError` is rare, 2) unlike `request.xxx`, `kwargs` is `dict`.

Reply:
... and that creates another problem to iterate while deleting. It's ok to create another dict if that's not solvable.